packages/cloud/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@logto/cloud",
+  "version": "0.1.0",
+  "description": "☁️ Logto Cloud service.",
```

Comment:
What is the triangle like symbol at the beginning? Is this on purpose?

Reply:
this is a cloud emoji actually - i think it reuses the encoding for a "cloud symbol"